lib/templates.ts

```diff
@@ -604,27 +604,38 @@ export async function updateIsPublishedForTemplate(
   publishFormType: string,
   publishDescription: string
 ): Promise<FormRecord | null> {
+  // Alias the isPublished value to newPublishStatus for clarity within the function
```

Comment:
Didn't want to change the function signature for now but "isPublished" is confusing here as it's the incoming status and not the DB status for the template.